Makefile

```diff
@@ -1,21 +1,22 @@
 BUILD_DIR  ?= build
 GOX_ARCH   ?= darwin/386 darwin/amd64 linux/amd64 linux/386
 GOX_OUTPUT ?= $(BUILD_DIR)/{{.Dir}}_{{.OS}}_{{.Arch}}
+GOMODULE   ?= GO111MODULE=on
 
 default: install test build
 
 install:
-	go get -t -v ./...
-	go get github.com/mitchellh/gox
+	$(GOMODULE) go get -t -v ./...
```

Comment:
This step is not needed with go modules :)

Reply:
@mikkeloscar I will remove that with a sub-sequent PR